cmd/storj-sim/network.go

```diff
@@ -281,6 +285,9 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 				"--version.server-address", fmt.Sprintf("http://%s/", versioncontrol.Address),
 				"--debug.addr", net.JoinHostPort("127.0.0.1", port(satellitePeer, i, debugHTTP)),
+
+				"--database", satelliteDB,
+				"--metainfo.database-url", flags.MetainfoDB,
```

Comment:
These won't work with multiple satellites, you need to use schemas to make it work.